cert-manager-plugin/pkg/syncers/secrets/syncer.go

```diff
@@ -29,6 +33,29 @@ type secretSyncer struct {
 	physicalClient client.Client
 }
 
+func (s *secretSyncer) getCertVirtualName(certPhysicalName string) (string, error) {
```

Comment:
I'm not sure, but I think this function and function below could be transformed to single function `getObjectVirtualName` or something like that.

Reply:
when Oleg is back from holiday he may have thoughts, but for me this seems fine as two simple little functions like this!